examples/querying-with-streams/querying_with_streams.bal

```diff
@@ -1,13 +1,41 @@
+import ballerina/io;
+
 type Error error;
 
 type LS stream<string, Error?>;
 
+type ValueRecord record {|
+    string value;
+|};
+
+const SAMPLE_LINE_COUNT = 5;
+
+class LineGenerator {
+    int i = -1;
+    string inputString;
+
+    public function init(string str) {
+        self.inputString = str;
+    }
+
+    public isolated function next() returns ValueRecord|Error? {
```

Comment:
Dont we need to add a code comments explaining `next` method?

Reply:
We have a separate bbe for Stream type.
https://dev.ballerina.io/learn/by-example/stream-type